{"uuid":"bbe67e9a-e2a1-4587-b620-bd0b579fc3ef","name":"Test case 8: should throw a TypeError if input is a number","historyId":"b80d9a135ea978533df503f863746597:d41d8cd98f00b204e9800998ecf8427e","status":"passed","statusDetails":{},"stage":"finished","steps":[],"attachments":[],"parameters":[],"labels":[{"name":"language","value":"javascript"},{"name":"framework","value":"jest"},{"name":"package","value":"functionsUnittests.objectFunctions.safeSet.test.ts"},{"name":"host","value":"pkrvmfkh1xnbeme"},{"name":"thread","value":"2"},{"name":"parentSuite","value":"safeSet"}],"links":[],"start":1752001510249,"fullName":"functionsUnittests/objectFunctions/safeSet.test.ts#safeSet Test case 8: should throw a TypeError if input is a number","testCaseId":"b80d9a135ea978533df503f863746597","stop":1752001510256}